core/controllers/editor.py

```diff
@@ -624,9 +600,9 @@ def get(self, exploration_id, escaped_state_name):
 
             try:
                 # Normalize the answers.
-                for answer in answers:
-                    answer['value'] = interaction_instance.normalize_answer(
-                        answer['value'])
+                for answer in submitted_answers:
+                    answer['answer'] = interaction_instance.normalize_answer(
```

Comment:
I agree this needs to be resolved, but I feel like it should be done after #1205 is submitted to avoid any risks of regressions. #3289 is filed to track and prioritize this work.